i6_models/assemblies/conformer/conformer.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+__all__ = ["ConformerBlockV1Config", "ConformerEncoderV1Config", "ConformerBlockV1", "ConformerEncoderV1"]
+
+import torch
+from torch import nn
+from dataclasses import dataclass
+
+from i6_models.config import ModelConfiguration, SubassemblyWithOptions
+from i6_models.parts.conformer import (
+    ConformerConvolutionV1,
+    ConformerConvolutionV1Config,
+    ConformerMHSAV1,
+    ConformerMHSAV1Config,
+    ConformerPositionwiseFeedForwardV1,
+    ConformerPositionwiseFeedForwardV1Config,
+)
+
+
+@dataclass
+class ConformerBlockV1Config(ModelConfiguration):
+    """
+    Attributes:
+        ff_cfg: Configuration for ConformerPositionwiseFeedForwardV1
+        mhsa_cfg: Configuration for ConformerMHSAV1
+        conv_cfg: Configuration for ConformerConvolutionV1
+    """
+
+    # nested configurations
+    ff_cfg: ConformerPositionwiseFeedForwardV1Config
+    mhsa_cfg: ConformerMHSAV1Config
+    conv_cfg: ConformerConvolutionV1Config
```

Comment:
Do we really want the `_cfg` postfix always? I think it is somewhat obsolete. Whatever we do, I think it should be consistent. See also my other comment.

Reply:
I am not 100% sure about this. So I agree here this is somehwat redundant, so my first instinct would be to remove it too. But on the other hand these configs are also meant as blueprints for other configs where maybe other ff parameters might be relevant (im making up params now but e.g. ff_count or ff_ordering). And then I am not sure if we want to have one "ff" param and the other two ones with some postfix. 
But I am not really decided on this.

Reply:
I'm not really sure I understand your argument pro having the `_cfg` postfix. Or rather, I don't really see why that is a problem.